plone/app/blocks/utils.py

```diff
@@ -69,9 +69,31 @@ def extractCharset(response, default='utf-8'):
 def resolve(url, resolved=None):
     """Resolve the given URL to an lxml tree.
     """
-
     if resolved is None:
-        resolved = resolveResource(url)
+        try:
+            resolved = resolveResource(url)
+        except Exception as e:
+            logger.error(
```

Comment:
I believe this should be logger.exception to not swallow the stacktrace.

Otherwise LGTM.